### PR TITLE
MQTT: Document Java Config

### PR DESCRIPTION
**Pull request by artembilan**
http://stackoverflow.com/questions/33655970/eclipse-paho-mqtt-spring-java-configuration

### CODE REVIEW

File: src/reference/asciidoc/mqtt.adoc [hunk 1]
```diff
@@ -101,6 +101,54 @@ The changes are not retained beyond the life cycle of the application context; a
...
+    			                                 "topic1", "topic2");
+    	adapter.setCompletionTimeout(5000);
+    	adapter.setConverter(new DefaultPahoMessageConverter());
+    	adapter.setQos(1);
+    	adapter.setOutputChannel(mqttInputChannel());
+    	return adapter;
+    }
+
+    @Bean
+    @ServiceActivator(inputChannel = "amqpInputChannel")
```
Comment (garyrussell) [inline src/reference/asciidoc/mqtt.adoc]: amqp?

**Comment by garyrussell** [comment]
Will fix on merge.

File: src/reference/asciidoc/mqtt.adoc (hunk 1 above)
Comment (artembilan) [inline src/reference/asciidoc/mqtt.adoc]: No, sorry.
Copy/paste legacy :smile:
Will appreciate if you fix on merge.
If it is only concern, of course...

File: src/reference/asciidoc/mqtt.adoc [hunk 2]
```diff
@@ -165,3 +213,55 @@ Default: `false`.
...
+        DefaultMqttPahoClientFactory factory = new DefaultMqttPahoClientFactory();
+        factory.setServerURIs("tcp://host1:1883", "tcp://host2:1883");
+        factory.setUserName("username");
+        factory.setPassword("password");
+        return factory;
+    }
+
+    @Bean
+    @ServiceActivator(inputChannel = "mqttOutboundChannel")
+    public MessageHandler amqpOutbound() {
```
Comment (garyrussell) [inline src/reference/asciidoc/mqtt.adoc]: ditto

**Comment by garyrussell** [comment]
Merged as febea4fe4020d2fe088aed44ca3f9731ad44f403

**Pull request closed without merging**